app/models/user.rb

```diff
@@ -34,7 +36,24 @@ class User < ActiveRecord::Base
   has_many :status
 
   accepts_nested_attributes_for :status
-  scope     :mail_receiver, -> { where(receive_mailings: true) }
+  scope :mail_receiver, -> { where(receive_mailings: true) }
+  scope :project_filter, -> (project_id) {
+    joins(:follows)
+    .where(
+      follows: {
+        blocked: false,
+        followable_id: project_id,
+        followable_type: 'Project',
+        follower_type: 'User'
+      }
+    )
+  }
+  scope :timezone_filter, -> (degrees) {
+    where("longitude BETWEEN ? AND ?", degrees[0], degrees[1])
+  }
+  scope :allow_to_display, -> { where(display_profile: true) }
+  scope :by_create, -> { order(:created_at) }
+  scope :online, -> (argument) { where("updated_at > ?", 10.minutes.ago) }
```

Comment:
I think all the scopes should be given names that make it obvious that they are scopes and not some method, e.g. name them all `by_....`